packages/components/addon/components/hds/alert/index.hbs

```diff
@@ -0,0 +1,24 @@
+<div class={{this.classNames}} ...attributes role="alert">
+
+  {{#if @icon}}
+    <div class="hds-alert__icon">
+      <FlightIcon
+        @name={{this.icon}}
+        @size="24"
+        {{! TODO: Replace w/dynamic color, once color prop implemented }}
+        @color="var(--token-color-foreground-warning-on-surface)"
+        @isInlineBlock={{false}}
+      />
+    </div>
+  {{/if}}
+
+  <div class="hds-alert__text">
+    {{#if @title}}
+      <p class="hds-alert__title">{{this.title}}</p>
+    {{/if}}
+
+    {{#if @body}}
```

Comment:
just a reminder, we agreed to call this `description`

Reply:
https://github.com/hashicorp/design-system/pull/92/commits/79543ef6be094df6d793f81b2df942ddecdfef26

will have more of what we talked about in new branch before monday